test-zone-data/main.cfg

```diff
@@ -0,0 +1,45 @@
+# This is the main coredns configuration to be loaded
+
+# root-zone on two "servers"
+. {
+    bind 127.1.0.1
+    bind fda1:b2:c3::127:1:0:1
+    file COMMON/root-zone . 
+    file COMMON/arpa arpa
+}
+
+. {
+    bind 127.1.0.2
+    bind fda1:b2:c3::127:1:0:2
+    file COMMON/root-zone .
+    file COMMON/arpa arpa
+}
+
+# internal resolver on one "server
```

Comment:
`"server` -> `"server"`

Reply:
I removed the quotes instead, also above.